generator/src/lib.rs

```diff
@@ -3019,18 +3018,32 @@ pub fn write_source_code<P: AsRef<Path>>(vk_headers_dir: &Path, src_dir: P) {
         #(#aliases)*
     };
 
-    write!(&mut vk_features_file, "{feature_code}").expect("Unable to write vk/features.rs");
-    write!(&mut vk_definitions_file, "{definition_code}")
-        .expect("Unable to write vk/definitions.rs");
-    write!(&mut vk_enums_file, "{enum_code}").expect("Unable to write vk/enums.rs");
-    write!(&mut vk_bitflags_file, "{bitflags_code}").expect("Unable to write vk/bitflags.rs");
-    write!(&mut vk_constants_file, "{constants_code}").expect("Unable to write vk/constants.rs");
-    write!(&mut vk_extensions_file, "{extension_code}").expect("Unable to write vk/extensions.rs");
-    write!(&mut vk_feature_extensions_file, "{feature_extensions_code}")
-        .expect("Unable to write vk/feature_extensions.rs");
-    write!(&mut vk_const_debugs_file, "{const_debugs}")
-        .expect("Unable to write vk/const_debugs.rs");
-    write!(&mut vk_aliases_file, "{aliases}").expect("Unable to write vk/aliases.rs");
+    fn write_formatted(text: &[u8], out: File) {
+        let mut child = std::process::Command::new("rustfmt")
+            .stdin(std::process::Stdio::piped())
+            .stdout(out)
+            .spawn()
+            .expect("Failed to spawn `rustfmt`");
+        let mut stdin = child.stdin.take().expect("Failed to open stdin");
+        stdin.write_all(text).unwrap();
+        drop(stdin);
+
+        let status = child.wait().unwrap();
+        assert!(status.success());
```

Comment:
Probably even faster if we batch this wait for all the files below... But that's a bit over the top.

Reply:
```
Benchmark 1: ./target/release/generator
  Time (mean ± σ):      2.165 s ±  0.041 s    [User: 1.795 s, System: 0.363 s]
  Range (min … max):    2.090 s …  2.218 s    10 runs
```

```
Benchmark 1: ./target/release/generator
  Time (mean ± σ):      1.577 s ±  0.025 s    [User: 1.784 s, System: 0.369 s]
  Range (min … max):    1.532 s …  1.612 s    10 runs
```

Trivial and significant wins after all.